README.md

```diff
@@ -39,16 +45,26 @@ We also have a set of [examples](https://github.com/beehive-lab/TornadoVM/tree/m
 
 [Profiler Usage](assembly/src/docs/9_PROFILER.md)
 
-
 ## 3. Programming Model
 
-TornadoVM exposes to the programmer task-level, data-level and pipeline-level parallelism via a light Application Programming Interface (API). TornadoVM uses single-source property, in which the code to be accelerated and the host code live in the same Java program.
+TornadoVM exposes to the programmer task-level, data-level and pipeline-level parallelism via a light Application
+Programming Interface (API). TornadoVM uses single-source property, in which the code to be accelerated and the host
+code live in the same Java program.
+
+Compute-kernels in TornadoVM can be programmed using two different APIs:
```

Comment:
the API is the same right? two different ways or approaches?

Reply:
Correct. I will fix this.